mula/scheduler/connectors/listeners/listeners.py

```diff
@@ -56,32 +67,53 @@ def __init__(self, dsn: str):
                 connect to.
         """
         super().__init__()
-        self.dsn = dsn
 
+        self.dsn = dsn
+        self.executor: futures.ThreadPoolExecutor = futures.ThreadPoolExecutor(max_workers=10)
         self.connection = pika.BlockingConnection(pika.URLParameters(self.dsn))
         self.channel = self.connection.channel()
+        self.is_running = True
```

Comment:
Isn't this `True` only during the loop in `self.basic_consume`?

Reply:
Or is this more of an `is_connected`?

Reply:
Fixed